source/views/directory/query.ts

```diff
@@ -0,0 +1,59 @@
+import ky from 'ky'
+import {useQuery} from '@tanstack/react-query'
+import {DirectorySearchTypeEnum, SearchResults} from './types'
+
+let directory = ky.create({prefixUrl: 'https://www.stolaf.edu/directory'})
+
+const getDirectoryParams = (query: string, type: DirectorySearchTypeEnum) => {
+	let p = new URLSearchParams()
+	p.set('format', 'json')
+
+	switch (type) {
+		case 'department':
+			p.set('department', query.trim())
```

Comment:
What are your thoughts on calling query.trim() once before the switch/case?

Reply:
I'd be down for that! They do all share "query", after all